src/pages/registration/contributors_tab/components/AddContributorModalContent.tsx

```diff
@@ -108,12 +108,45 @@ export const AddContributorModalContent = ({
       {isLoadingAuthorities ? (
         <PageSpinner />
       ) : authorities && authorities.length > 0 && debouncedSearchTerm ? (
-        <AuthorityList
-          authorities={authorities}
-          selectedArpId={selectedAuthor?.id}
-          onSelectAuthority={setSelectedAuthor}
-          searchTerm={debouncedSearchTerm}
-        />
+        authorities.length > 6 ? (
```

Comment:
syns egentlig det er litt rart om den av og til dukker opp øverst. hvis vi skal ha det burde det dras ut som egen komponent da, så vi slipper duplisering av kode. men personlig heller jeg mot å bare ha det nederst

men ser også at det blir uoversiktlig når lista blir lang og man må scrolle for å finne actions :/ skulle kanskje helst hatt paging her?

Reply:
Ja, det er ikke helt 100 sånn der hvertfall. Har sendt link til skjermbildene til Kine også, så kan hun se på det. Men paging er kanskje det beste ja

Reply:
liten preview av dette inkl. valg av type: 
![bilde](https://user-images.githubusercontent.com/6313468/115246450-31b97900-a126-11eb-8ba3-0438cea21ec8.png)

mulig vi kan sette bredden på kolonnene litt smartere også, virker unødvendig at alle brukere får flere linjer bare fordi noen trenger det.

Reply:
ser ut som backend bare returnerer 10 treff, uten noen mulighet for paging atm, så det får kanskje bare ligge for nå. kan evt opprette en oppgave på det

Reply:
brukte truncate på almatittel nå

Reply:
Måtte sjekke mot denne: https://unit.atlassian.net/browse/NP-2456 :P

![bilde](https://user-images.githubusercontent.com/6313468/115548602-efb84080-a2a7-11eb-91f4-cf920dba75ef.png)

Reply:
what? hvorfor skjer det der?

Reply:
skjønner ikke helt hvorfor det ikke fungerer, men løste seg med grid og dropping av span ala det du gjorde på den PRen der.

Reply:
truncate-pakken har litt snedig innpakking av `span`, så det hender at en av de blir veldig liten :/
Løsningen jeg fant var bare å sette bredde på span-elementer, ligger link til PRen i Jira-saken